commcare_connect/program/forms.py

```diff
@@ -48,3 +49,22 @@ def save(self, commit=True):
             instance.created_by = self.user.email
         instance.modified_by = self.user.email
         return super().save(commit=commit)
+
+
+class ManagedOpportunityInitForm(OpportunityInitForm):
+    class Meta(OpportunityInitForm.Meta):
+        model = ManagedOpportunity
+        fields = OpportunityInitForm.Meta.fields + [
+            "org_pay_per_visit",
```

Comment:
I think this should be set on the budget part of the creation workflow, not the initial page, since that is what it is contributing to.